bigbluebutton-html5/imports/startup/server/index.js

```diff
@@ -6,12 +6,24 @@ import './settings';
 import Logger from './logger';
 import Redis from './redis';
 
+var parse = Npm.require('url').parse;
+
 const AVAILABLE_LOCALES = fs.readdirSync('assets/app/locales');
+const CDN_URL = process.env.CDN_URL || '';
 
 Meteor.startup(() => {
   const APP_CONFIG = Meteor.settings.public.app;
   const env = Meteor.isDevelopment ? 'development' : 'production';
-  Logger.warn(`SERVER STARTED. ENV=${env}, nodejs version=${process.version}`, APP_CONFIG);
+
+  // Add CDN
+  WebAppInternals.setBundledJsCssPrefix(CDN_URL);
+
+  // Trust the URL in our browser policy (if it's available).
+  try {
+    return BrowserPolicy.content.allowOriginForAll(CDN_URL);
+  } catch (undefined) {}
```

Comment:
Is it possible to log the error that was in the `catch` block?

Reply:
I removed this line. We don't need it.